src/Exception/SnmpApiError.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SimPod\PhpSnmp\Exception;
+
+use RuntimeException;
+use SimPod\PhpSnmp\SnmpException;
+use function sprintf;
+
+final class SnmpApiError extends RuntimeException implements SnmpException
+{
+    public static function invalidJson(string $error) : self
+    {
+        return new self(sprintf('Failed to decode JSON response: %s', $error));
```

Comment:
I don't see how is this better ... the first one is more readable I think... it will also look much better in case of multi-line errors

Reply:
My idea here is that it might not be clear from the message when the actual response starts. We don't know in advance what comes so the response content should be somehow surrounded by quotes, braces etc.

Reply:
I don't see any reason to do so...What does the separation brings to the table? I prefer messages that are easy to read.

Reply:
And that's it, it's not easy to read if it's not easy to distinguish whether the part of message is the response already or not